org.eclipse.ice.commands/src/main/java/org/eclipse/ice/commands/RemoteCommand.java

```diff
@@ -25,7 +36,17 @@
 	 * The particular connection associated to a particular RemoteCommand. Declare
 	 * this up front since by definition a RemoteCommand must have a connection.
 	 */
-	Connection connection = new Connection();
+	protected Connection connection = new Connection();
+
+	/**
+	 * A file output stream for error messages to be remotely logged to
+	 */
+	FileOutputStream stdErrStream = null;
+
+	/**
+	 * A file output stream for output messages to be remotely logged to
+	 */
+	FileOutputStream stdOutStream = null;
```

Comment:
Should be private/protected

Reply:
@SmithRWORNL How did you do quoted comment of a tiny piece of code? Awesome!

Reply:
@jayjaybillings Go into Files Changed, hover over the lines you want, and click and drag.